### DISCUSSION TITLE
package.json not being detected in a project that also has yarn workspaces

### DISCUSSION THREAD

**Discussion by deviantintegral** [renovatebot/renovate#22273] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

We have a repository with Yarn with the following structure:

```
package.json
themes/project/package.json
tests/playwright/package.json
```

The root package.json contains a workspace that includes `themes/project/package.json`. However, `tests/playwright/package.json` is a totally separate project and is not contained in the workspace. This is mainly because many of our team members don't need Playwright, and pulling it in is a significant download.

Under `npm` in the dependency dashboard, only the first two package.json files are detected.

### What you tried so far.

We thought we could add the following to `renovate.json`. It doesn't show any errors, but also doesn't work. Nothing in the logs references test/playwright/package.json.
```
{
  "packageRules": [
    {
      "paths": [
        "test/playwright/package.json"
      ]
    }
  ]
}
```

How can we get Renovate to work with this layout? https://docs.renovatebot.com/modules/manager/npm/ says "By default, Renovate will check any files matching the following regular expression: (^|/)package\.json$.". Perhaps workspaces are entirely unrelated, but this it's the only difference I can think of.

**Comment by rarkins** [comment, marked as answer]
Test folders are ignored as part of the config:base preset you use

**Reply by deviantintegral** [reply]
I would not have guessed that on my own.

For anyone else, this is shown the logs:

```
DEBUG: Found repo ignorePaths
{
  "ignorePaths": [
    "**/node_modules/**",
    "**/bower_components/**",
    "**/vendor/**",
    "**/examples/**",
    "**/__tests__/**",
    "**/test/**",
    "**/tests/**",
    "**/__fixtures__/**"
  ]
}
```

This is documented at: https://docs.renovatebot.com/presets-default/#ignoremodulesandtests

Once we overrode ignorePaths and removed `**/test/**`` the updates were picked up. Thank you!
